apps/email/style/compose_cards.css

```diff
@@ -88,6 +88,58 @@ input.cmp-subject-text {
   text-overflow: ellipsis;
 }
 
+/* Allows the position absolute autocomplete list to be relative to this
+   element */
+.cmp-autocomplete-origin {
+  position: relative;
+}
+
+/* Prevents scrolling of areas under the autocomplete list to avoid confusing
+   dueling scrollbars */
+.cmp-autocomplete-noscroll {
+  overflow: hidden;
+}
+
+/* When the autocomplete is active, change the flex alignment so that the label
+and + buttons are visible when typing */
+.cmp-autocomplete-noscroll .cmp-envelope-line {
+  align-items: flex-end;
+}
+
+.cmp-autocomplete-spacer {
+  background-color: #f2f2f2;
```

Comment:
That sounds good. We pull in shared/elements/gaia-theme/gaia-theme.css because of a shared dependency. That has some vars defined, but it does not include this particular color. I expect if we revisit the UI for making things more common across apps or a general visual style change we will do that then.